cocoa/src/toga_cocoa/container.py

```diff
@@ -67,7 +67,7 @@ def __init__(
             NSLayoutAttributeLeft,
             1.0,
             min_width,
-        )
+        ).retain()
```

Comment:
For future reference, just adding a link to https://github.com/beeware/rubicon-objc/issues/256.